xapian-core/matcher/extraweightpostlist.cc

```diff
@@ -39,7 +39,7 @@ ExtraWeightPostList::get_weight(Xapian::termcount doclen,
      * at some point, and so that we got all the incompatible changes needed to
      * add support for the number of unique terms over with in one go.
```

Comment:
This comment needs updating too...

Reply:
I have updated it. (Not sure if it's completely correct.)